pkg/policies/branch/branch.go

```diff
@@ -60,6 +61,14 @@ type OrgConfig struct {
 
 	// BlockForce : set to true to block force pushes, default true.
 	BlockForce bool `yaml:"blockForce"`
+
+	// RequireUpToDateBranch : set to true to require that branches must be up
+	// to date before merging. Default false.
+	RequireUpToDateBranch bool `yaml:"requireUpToDateBranch"`
```

Comment:
I'm thinking that this should default to true, but should only take effect if the RequireStatusChecks array length is non-zero.

Reply:
The GitHub UI text is: "This setting will not take effect unless at least one status check is enabled (see below)."

Reply:
Ah I totally missed that UI text. Thanks for pointing it out! I'll work on updating the logic to reflect your comment.

Reply:
Updated to only look at `RequireUpToDateBranch` if `RequireStatusChecks` is set: 754144bd3f9e24f3554793450990d44d4116b98f